### ISSUE TITLE
Error: HTTP 500 When URL is accessed, and you aren't logged in.

### ISSUE THREAD

**Issue by sn3ak** [antedebaas/DMARC-Reports#50]
Instead of redirecting to a login screen, going to a URL such as http://127.0.0.1/reports/smtptls?page=5&perpage=17 in a different browser/incognito results in the following error:

```
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
```

**Comment by antedebaas** [comment]
yeah ive seen this one. its on my list :)

**Comment by antedebaas** [comment]
fixed in 1.9.3
